XPath-CSS Project/css-answers.txt

```diff
@@ -0,0 +1,27 @@
+1) Page:/text-box. Write a selector that will return the ‘Full Name’ textbox using the ID attribute. 
+
+==> input#userName
+
+2) Page: /buttons. Write a selector that will return all 3 buttons shown below.
+
+==> button.btn-primary
+
+3) Page:/buttons. Write a selector that will return the span element in the footer.
+
+==> footer span
+
+4) Page:/menu.Write a selector that will return the 3 list item elements in the unordered list.
+
+==> ul#nav > li
+
+5) Page: /select-menu. Write a selector that will get the option element where value = 4.
+
+==> option[value='4']
```

Comment:
ensure that you are searching with the context of the ui control
div#selectMenuContainer option[value="4"]